client/cs/src/StateAccessors.cs

```diff
@@ -129,7 +129,7 @@ public States.State_ObjectHandle State(States.StateHandle_ObjectHandle state_han
             var p = new Ptr<States.State_ObjectHandle>(
                 ffi.TpClientBaselineBaselineStateObjectHandle(this.Inner.Value.p, state_handle.Inner.Value.p)
             );
-            return new States.State_ObjectHandle(p, OwnershipSemantics.SharedRef);
+            return new States.State_ObjectHandle(p, OwnershipSemantics.Owned);
```

Comment:
In rust, this is a reference: https://github.com/0xTELEPORTAL/constellation/blob/b67f6232ebe42da9dd05e6682a2c59e8dcfb2b21/client/rust/src/baseline.rs#L448-L456

So it must be `SharedRef`

Reply:
Good catch, updating.